lib/testmap.py

```diff
@@ -127,13 +127,15 @@ def contexts(image, *scenarios: Iterable[str]):
         ],
     },
     'cockpit-project/cockpit-navigator': {
+        'main': [
+            'fedora-38',
+            'fedora-39',
+            'fedora-rawhide',
```

Comment:
Missing `]`, which explains the unit test failure. You didn't actually run fedora-38 on our CI, but as it runs on packit, it should be fine.